test/python/transpiler/test_decompose.py

```diff
@@ -100,9 +133,9 @@ def test_decompose_oversized_instruction(self):
     def test_decomposition_preserves_qregs_order(self):
         """Test decomposing a gate preserves it's definition registers order"""
         qr = QuantumRegister(2, "qr1")
-        qc = QuantumCircuit(qr)
-        qc.cx(1, 0)
-        gate = qc.to_gate()
+        qc1 = QuantumCircuit(qr)
+        qc1.cx(1, 0)
+        gate = qc1.to_gate()
```

Comment:
Minor comment, but for the future it's easier to review if you only change the code corresponding to your feature/bugfix and leave the rest of the existing code as is 🙂